src/test/js/multibranch/noRunsActivityPage.js

```diff
@@ -0,0 +1,61 @@
+const jobName = 'noRunsActivityPage';
+const tmp = require('tmp');
+
+const repo = tmp.dirSync();
+const pathToRepo = repo.name;
+const path = require("path");
+const sourceRep = './src/test/resources/multibranch/test_results';
+const git = require("../../../main/js/api/git");
+
+
+/** @module failing
+ * @memberof testcases
+ * @description TEST: basic tests around the failing pipeline. Test whether the result is not collapsed.
+ */
+module.exports = {
+    // ** creating a git repo */
+    before: (browser, done) => {
+          // we creating a git repo in target based on the src repo (see above)
+          git.createRepo(sourceRep, pathToRepo).then(done);
+    },
+    /**
+     * Create Multibranch Pipeline Job "noRunsActivityPage" with no branches.
+     * @param browser
+     */
+    'Step 01 - Create job': (browser) => {
+        var multibranchCreate = browser.page.multibranchCreate().navigate();
+      
+        multibranchCreate.createBranch(jobName, pathToRepo);
+    },
+
+    'Step 02 - Delete runs': (browser) => {
+        browser.useXpath()
+            // Navidate back to main job screen
+            .waitForElementVisibleThenClick('//*[@id="breadcrumbs"]/li[3]/a')
+            // Navigate to master branch.
+            .waitForElementVisibleThenClick('//*[@id="job_master"]/td[3]/a')
+            // Wait until run is done. TODO make this better.
+            .pause(5000)
```

Comment:
Ah no ... please, no `pause` calls ... I'm not even going to look at the rest of this PR :)